src/run.py

```diff
@@ -9,12 +10,28 @@
 kwargs = get_parameter()
 
 # check if a toolname was set in env
-toolname = os.environ.get('TOOL_RUN', 'whitebox_info').lower()
+toolname = os.environ.get('TOOL_RUN', 'fill_depressions').lower()
 
 # switch the tool
 if toolname == 'whitebox_info':
     wblib.print_info(to_file=kwargs.get('toFile', True))
 
+ # Fill Depression tool
+elif toolname == 'fill_depressions':
+    # get the parameters
+    try:
+        inp = kwargs['inputDEM']
+        out = kwargs['outputDEM']
+        flats = kwargs['fix_flats']
```

Comment:
you can replace this line with:

```python
flats = kwargs.get('fix_flats'; True)
```

Then, if the parameter is missing, it will always be set to `True`, for convenience